openedx/core/djangoapps/auth_exchange/README.rst

```diff
@@ -0,0 +1,12 @@
+auth_exchange
+-------------
+
+Views to support exchange of authentication credentials.
+The following are currently implemented::
```

Comment:
```suggestion
The following are currently implemented:
```

Double-colon isn't needed here because there's no code block that will be indented.